libs/member-plan/api/src/lib/member-plan.model.ts

```diff
@@ -32,7 +33,7 @@ export class MemberPlan {
   @Field()
   description!: string
 
-  @Field()
+  @FieldProtected(() => Boolean, {permissions: [CanGetMemberPlan]})
```

Comment:
Can probably always be returned and only query active memberplans for users with the perms instead

Reply:
I wonder if we can introspect the type for boolean, number etc, so we don't always have to specify.

Reply:
Agree, it's not really harmful to have it.